modules/installation-obtaining-installer.adoc

```diff
@@ -99,9 +99,9 @@ ifdef::openshift-origin[]
 +
 If you do not use the pull secret from the {cloud-redhat-com} site:
 +
-* Red Hat Operators are not available. 
+* Red Hat Operators are not available.
 * The Telemetry and Insights operators do not send data to Red Hat.
-* Content from the link:https://registry.redhat.io/[Red Hat Container Catalog] registry, such as image streams and Operators, are not available. 
+* Content from the link:https://registry.redhat.io/[Red Hat Ecosystem Catalog Container images] registry, such as image streams and Operators, are not available. 
```

Comment:
Should this link follow the other changes and point to https://catalog.redhat.com/software/containers/explore?

Reply:
That's a good question. registry.redhat.io redirects to https://catalog.redhat.com/software/containers/explore. I can go ahead and change those, but I think I'll need to open a CIT issue to change registry.redhat.io links to https://catalog.redhat.com/software/containers/explore in other files because there are a lot of independent references and links to the registry.redhat.io URL.